### DISCUSSION TITLE
Media Server Integration

### DISCUSSION THREAD

**Discussion by unclebearr** [Geoxor/Amethyst#608] [Ideas]
Ability to add media servers such as Plex, Jellyfin, etc.
Maybe even other network file protocols like sftp, webdav.. idk

Project focused on that aspect strictly for Jellyfin:
https://github.com/jeffvli/sonixd

**Comment by freqawi** [comment]
How would adding plex or jellyfin support do anything?

Amethyst is a media player just like jellyfin.. it wouldn't really do anything I don't think nor would it even be approachable.. 

As for that kind of concept, you can just open a folder on a nas or network drive and it should work just the same.

Edit: I know jellyfin isn't a 'player' but you get my point.. you can still just open a music folder from a network drive.
